platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiBasicActionMoveMoveLayoutTestIT.java

```diff
@@ -0,0 +1,208 @@
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.ELEMENT_MOVE_PAGE_DESTINATION_LABEL;
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.ELEMENT_MOVE_PAGE_SELECT_THE_DESTINATION_LABEL;
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.ELEMENT_MOVE_SPACESWITCHER;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+
+@Tag("smoke")
+@Tag("wiki")
+public class WikiBasicActionMoveMoveLayoutTestIT extends Base {
+
+    HomePagePlatform homePagePlatform;
+    SpaceManagement spaceManagement;
+    WikiHomePage wikiHomePage;
+    WikiManagement wikiManagement;
+    RichTextEditor richEditor;
+    SpaceHomePage spaceHomePage;
+
+
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+
+        homePagePlatform = new HomePagePlatform(this);
+        spaceHomePage = new SpaceHomePage(this);
+        wikiHomePage = new WikiHomePage(this);
+        wikiManagement = new WikiManagement(this);
+        spaceManagement = new SpaceManagement(this);
+        richEditor = new RichTextEditor(this);
+    }
+
+    @Test
+    public void test01_MoveLayoutIsDisplayingSelectTheDestiNationLabel() {
+        String space1 = "space1" + getRandomNumber();
+        String title = "title" + getRandomNumber();
+        String content = "content" + getRandomNumber();
+        info("Create data test");
+        info("Create Space 1 with a wiki page 1");
+        homePagePlatform.goToMySpaces();
+        spaceManagement.addNewSpaceSimple(space1, space1);
+
+        /*Step Number: 1
+         *Step Name: Step 1
+		 *Step Description:
+		 *- Go to the wiki of the "Space Move"
+		 *Input Data:
+         *Expected Outcome:
+	     *- Wiki is displayed*/
+		/*Step Number: 2
+		 *Step Name: Step 2
+		 *Step Description:
+		 *- Create a new Page with the title Page A
+		 *- Save
+		 *Input Data:
+         *Expected Outcome:
+		 *- Page A is displayed*/
+        info("Create wiki page in sapce1");
+        spaceHomePage.goToWikiTab();
+        wikiHomePage.goToAddBlankPage();
+        richEditor.addSimplePage(title, content);
+        wikiManagement.saveAddPage();
+
+        /*Step Number: 3
+		 *Step Name: Step 3
+		 *Step Description:
+		 *- Click on [More] on menu
+		 *Input Data:
+         *Expected Outcome:
+		 *- Move option is available*/
+        wikiHomePage.goToMovePageForm();
+
+		/*Step Number: 4
+		 *Step Name: Step 4
+		 *Step Description:
+		 *- Click on Move options
+		 *Input Data:
+         *Expected Outcome:
+		 *- A popup is displayed with the space switcher
+		 *- Label "Select the destination:" is displayed before the space switcher*/
+        info("Test 1: Move Layout is displaying Select the destination label");
+        info("Label Select the destination is displayed before the space switcher");
+        $(ELEMENT_MOVE_PAGE_SELECT_THE_DESTINATION_LABEL).waitUntil(Condition.visible, Configuration.timeout);
```

Comment:
We should delete the wiki page at the end of the test. (same for the others tests)

Reply:
done